### PR TITLE
Drafted team agreement based on discussion thread

### PR DESCRIPTION
**Pull request by matthewcoole**
Based around discussion https://github.com/NERC-CEH/rse_group/discussions/23.

This is a very rough incorporation of what has been discussed. The language is intentionally simple to make it as accessible as possible to others, and is limited to a single page - currently it is one page when converted with `pandoc` to `docx`.

I think in various places we can maybe add a few more specifics and some refinements here and there, but generally I think it's a good idea to keep it to a single page (if we don't think something is important enough to find space for in one page then it probably isn't important enough).

Perhaps if there is more to add we could consider starting a longer form document that could be some kind of UKCEH RSE handbook?

### CODE REVIEW

File: team/agreement.md [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+- In line with UKCEH's values of excellence, integrity and teamwork, we seek to understand the scientific activities we support to help build communities between teams at UKCEH.
+
+## Team Structure
+- We are a small team of 6 people across 3 UKCEH sites.
+- We are a flat team in structure - nobody's work, voice or ideas are more important than anyone else's.
+
+## Management and Workflow   
+- We work on projects in accordance with our governance structure ()[Insert Link].
+- We operate in an agile way and are flexible and open to change in what we do and how we do things.
+- We continually look to adopt the latest software development tools and best practices in our workflow, including pair/mob programming and code reviews.
```
Comment (longr) [inline team/agreement.md:15]: Perhaps replace latest with most appropriate, or best suited?

File: team/agreement.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+- We help each other with our work and strive to collectively ensure all within the team are well supported.
+
+## Professional Development
+- We share knowledge with each other and everyone we work with. Continuous learning is critical to how we work.
+- We seek to find innovative ways to share our knowledge and enhance everyone's professional development in the organisation by building communities of practice and learning.
+- We ensure that we devote time each week to enhancing our skills through active learning, dissemination or contributing to open-source (Friday?).
```
Comment (mattjbr123) [inline team/agreement.md:21]: Appreciate devoting explicit time to this

**Comment by mattjbr123** [comment]
Honestly @matthewcoole it looks fine to me, nothing jumps out at me as glaringly wrong or missing.

**Review by metazool — APPROVED** [review]
Given we've all had a lot of input and accept this will be an iterative document, happy to merge as-is and refine

**Pull request merged**